test/e2e/common/util.go

```diff
@@ -889,6 +889,61 @@ func IsVolumePublished(uuid string) bool {
 	return true
 }
 
+func CheckNoVolumeResources() (bool, bool) {
+	logf.Log.Info("CheckNoVolumeResources")
+	foundResources := false
+	success := true
+
+	pvcs, err := gTestEnv.KubeInt.CoreV1().PersistentVolumeClaims("default").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Error(err, "CheckNoVolumeResources: list PVCs failed.")
+		success = false
+	} else if pvcs != nil && len(pvcs.Items) != 0 {
+		logf.Log.Info("CheckNoVolumeResources: found PersistentVolumeClaims")
+		foundResources = true
+	}
+
+	pvs, err := gTestEnv.KubeInt.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Error(err, "CheckNoVolumeResources: list PVs failed.")
+	} else if pvs != nil && len(pvs.Items) != 0 {
+		logf.Log.Info("CheckNoVolumeResources: found PersistentVolumes")
+		foundResources = true
+	}
+
+	msvGVR := schema.GroupVersionResource{
+		Group:    "openebs.io",
+		Version:  "v1alpha1",
+		Resource: "mayastorvolumes",
+	}
+
+	msvs, err := gTestEnv.DynamicClient.Resource(msvGVR).Namespace("mayastor").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Info("CheckNoVolumeResources: list MSVs failed.", "Error", err)
+		success = false
+	} else if msvs != nil && len(msvs.Items) != 0 {
+		logf.Log.Info("CheckNoVolumeResources: found MayastorVolumes")
```

Comment:
Please log what volumes you've found (I imagine this function will be useful for bigger tests with multiple volumes and it will avoid guessing when failing).

Reply:
Done